website/content/en/docs/building-operators/golang/tutorial.md

```diff
@@ -270,36 +292,30 @@ The `ClusterRole` manifest at `config/rbac/role.yaml` is generated from the abov
 $ make manifests
 ```
 
-## Build and run the operator
+## Configuring your test environment
```

Comment:
it should be removed soon. However, until we are able to remove it, in this pr we are just fixing the step.